plugins/woocommerce/src/StoreApi/Authentication.php

```diff
@@ -75,10 +64,12 @@ public function exposed_cors_headers( $exposed_headers ) {
 	 *
 	 * Users of valid Cart Tokens are also allowed access from any origin.
 	 *
-	 * @param bool $value  Whether the request has already been served.
+	 * @param bool             $value  Whether the request has already been served.
+	 * @param \WP_REST_Server  $server The REST server instance.
+	 * @param \WP_REST_Request $request The REST request instance.
 	 * @return bool
 	 */
-	public function send_cors_headers( $value ) {
+	public function send_cors_headers( $value, $server, $request ) {
```

Comment:
@mikejolley just double checking these params against the `apply_filters` implementation [here](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/class-wp-rest-server.php#L515).

Looks like the second param is passed as a `WP_HTTP_Response` object so using `$server` appears misleading if I'm understanding this correctly, even if we overwrite it later anyway this still could potentially cause confusion.

Also, is there an opportunity to skip `$server = rest_get_server();` since `$server` is passed as the 4th param to the filter?

Reply:
Yeah you are correct. Fixed in https://github.com/woocommerce/woocommerce/pull/57963